src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,54 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION_TYPE = 0;
+    private static final int AMOUNT = 1;
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String content = readFile(fromFileName);
+        writeFile(toFileName, content);
+    }
+
+    private String readFile(String fileName) {
+        int supplyAmount = 0;
+        int buyAmount = 0;
+
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName))) {
+            String line;
+            while ((line = bufferedReader.readLine()) != null) {
+                String[] info = line.split(",");
+                if (info[OPERATION_TYPE].equals(BUY)) {
+                    buyAmount += Integer.parseInt(info[AMOUNT]);
+                } else if (info[OPERATION_TYPE].equals(SUPPLY)) {
+                    supplyAmount += Integer.parseInt(info[AMOUNT]);
+                }
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("File not found " + e);
```

Comment:
You are throwing a RuntimeException with a message that is not very informative. It's better to include more details about the error, for instance, the name of the file that caused the error. Also, you are concatenating the exception `e` directly into the message. It's better to pass it as a second argument to the RuntimeException constructor to keep the original stack trace. This is a code quality issue.